### PR TITLE
gh-1291 Added happy case

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Handles #1291 

- First happy case using as much of existing code as possible and JSON data for the test.

### CODE REVIEW

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift [hunk 1]
```diff
@@ -17,8 +17,7 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         case total(text: String)
         case balances(items: [TokenBalance])
     }
-
-    var clientGatewayService = App.shared.clientGatewayService
+    var clientGatewayService: BalancesAPI = App.shared.clientGatewayService
```
Comment (sche) [inline Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift:20]: 😅

File: MultisigIntegrationTests/CoreData/CoreDataTestCase.swift [hunk 2]
```diff
@@ -17,4 +17,29 @@ class CoreDataTestCase: XCTestCase {
         App.shared.coreDataStack = TestCoreDataStack()
         context = App.shared.coreDataStack.viewContext
     }
+
+    @discardableResult
+    func createSafe(name: String, address: String, network: Network = Network.mainnetChain()) -> Safe {
```
Comment (sche) [inline MultisigIntegrationTests/CoreData/CoreDataTestCase.swift:22]: 👍🏻

File: MultisigIntegrationTests/UI Snapshot Tests/BalancesViewControllerSnapshotTests.swift [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        // balances screen shows 'import key banner' unless already shown or imported key
+        AppSettings.importKeyBannerWasShown = true
+
+        // balances screen shows 'passcode banner' unless already shown or has key and no passcode is set
+        AppSettings.passcodeBannerDismissed = true
+
+        let vc = BalancesViewController()
+        vc.clientGatewayService = apiStub
+
+
```
Comment (sche) [inline MultisigIntegrationTests/UI Snapshot Tests/BalancesViewControllerSnapshotTests.swift:38]: why so much space?

**Review by sche — APPROVED** [review]
👍🏻

File: MultisigIntegrationTests/UI Snapshot Tests/BalancesViewControllerSnapshotTests.swift (hunk 3 above)
Comment (DmitryBespalov) [reply MultisigIntegrationTests/UI Snapshot Tests/BalancesViewControllerSnapshotTests.swift:38]: to separate 3 sections: set up, act, verify

**Pull request merged**